### ISSUE TITLE
Field options are striped when using fmt

### ISSUE THREAD

**Issue by rogchap** [emicklei/proto-contrib#7]
```
int32 age = 1 [(validator.field) = { int_gt: 0 }];
```

turns into 
```
int32 age = 1 [(validator.field) = ];
```

**Comment by emicklei** [comment]
i guess you mean when formatting?  thank you for reporting this, I will add a test and fix for it.

**Comment by kalbasit** [comment]
@emicklei has this been closed in 6528e79?
